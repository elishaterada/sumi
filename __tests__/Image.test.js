import React from 'react';
import renderer from 'react-test-renderer';
import { createSerializer } from 'jest-emotion';
import * as emotion from 'emotion';

import {
  ImageClass,
  ImageCircleClass,
} from '../src/components/Image';

expect.addSnapshotSerializer(createSerializer(emotion));

test('renders Image', () => {
  const reactComponent = renderer
    .create(
      <div>
        <img
          className={ImageClass}
          src="https://source.unsplash.com/ngcX09jzLE0/800x533"
          alt="Jelly Fish"
        />
        <img
          className={ImageCircleClass}
          src="https://source.unsplash.com/ngcX09jzLE0/800x533"
          alt="Jelly Fish"
        />
      </div>,
    )
    .toJSON();

  expect(reactComponent).toMatchSnapshot();
});
