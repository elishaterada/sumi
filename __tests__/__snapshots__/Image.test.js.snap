// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders Image 1`] = `
.emotion-0 {
  -webkit-animation-duration: 1s;
  animation-duration: 1s;
  -webkit-animation-fill-mode: both;
  animation-fill-mode: both;
  -webkit-animation-name: animation-5j8bii;
  animation-name: animation-5j8bii;
  max-width: 100%;
  border-radius: 0.1875rem;
}

.emotion-1 {
  -webkit-animation-duration: 1s;
  animation-duration: 1s;
  -webkit-animation-fill-mode: both;
  animation-fill-mode: both;
  -webkit-animation-name: animation-5j8bii;
  animation-name: animation-5j8bii;
  max-width: 100%;
  max-height: 100%;
  border-radius: 50%;
}

<div>
  <img
    alt="Jelly Fish"
    className="emotion-0"
    src="https://source.unsplash.com/ngcX09jzLE0/800x533"
  />
  <img
    alt="Jelly Fish"
    className="emotion-1"
    src="https://source.unsplash.com/ngcX09jzLE0/800x533"
  />
</div>
`;
